src/main/java/org/prebid/server/bidder/videobyte/VideobyteBidder.java

```diff
@@ -0,0 +1,155 @@
+        }
+    }
+
+    private HttpRequest<BidRequest> createRequest(BidRequest bidRequest, Imp imp, ExtImpVideobyte extImpVideobyte)
+            throws PreBidException {
+
+        final BidRequest modifiedBidRequest = bidRequest.toBuilder().imp(Collections.singletonList(imp)).build();
+        return HttpRequest.<BidRequest>builder()
+                .method(HttpMethod.POST)
+                .uri(createUri(extImpVideobyte))
+                .headers(headers(modifiedBidRequest))
+                .body(mapper.encodeToBytes(modifiedBidRequest))
+                .payload(modifiedBidRequest)
+                .build();
+    }
+
+    private String createUri(ExtImpVideobyte extImpVideobyte) throws PreBidException {
+        try {
+            final URIBuilder uriBuilder = new URIBuilder(endpointUrl)
+                    .addParameter("source", "pbs")
+                    .addParameter("pid", extImpVideobyte.getPublisherId());
+            letIfNotBlank(extImpVideobyte.getPlacementId(), it -> uriBuilder.addParameter("placementId", it));
+            letIfNotBlank(extImpVideobyte.getNetworkId(), it -> uriBuilder.addParameter("nid", it));
+
+            return uriBuilder.build().toString();
+        } catch (URISyntaxException e) { // shouldn't really happen
+            throw new PreBidException(e.getMessage());
+        }
+    }
+
+    private static MultiMap headers(BidRequest bidRequest) {
+        final MultiMap headers = HttpUtil.headers();
+
+        final Site site = bidRequest.getSite();
+        if (site != null) {
+            letIfNotBlank(site.getDomain(), it -> headers.add(HttpUtil.ORIGIN_HEADER, it));
+            letIfNotBlank(site.getRef(), it -> headers.add(HttpUtil.REFERER_HEADER, it));
+        }
+
+        return headers;
+    }
+
+    private static void letIfNotBlank(String value, Consumer<String> action) {
+        if (StringUtils.isNotBlank(value)) {
+            action.accept(value);
+        }
+    }
+
+    @Override
+    public Result<List<BidderBid>> makeBids(HttpCall<BidRequest> httpCall, BidRequest bidRequest) {
+        try {
+            final BidResponse bidResponse = mapper.decodeValue(httpCall.getResponse().getBody(), BidResponse.class);
+            return Result.withValues(extractBids(httpCall.getRequest().getPayload(), bidResponse));
+        } catch (DecodeException e) {
+            return Result.withError(BidderError.badServerResponse("Bad server response."));
+        }
+    }
+
+    private static List<BidderBid> extractBids(BidRequest bidRequest, BidResponse bidResponse) {
+        if (bidResponse == null || CollectionUtils.isEmpty(bidResponse.getSeatbid())) {
+            return Collections.emptyList();
+        }
+
+        final Map<String, BidType> impIdToMediaType = new HashMap<>();
+        for (Imp imp : bidRequest.getImp()) {
+            final BidType bidType = imp.getBanner() != null ? BidType.banner : BidType.video;
+            impIdToMediaType.put(imp.getId(), bidType);
+        }
```

Comment:
Pls, use project's default solution.

Reply:
Current solution similar to go code. Just for lite understanding when comparing code with go version.